.travis.yml

```diff
@@ -6,6 +6,8 @@ install:
     - pip install 'cwltool==1.0.20181217162649'
     - pip install 'ruamel.yaml==0.15.77'
     - pip install 'mdutils==1.0.0'
+    - pip install 'mdutils==1.0.0'
```

Comment:
This seems to be an accidental duplication.

Reply:
Good catch, fixed